python/cudf/cudf/core/dataframe.py

```diff
@@ -5481,6 +5481,9 @@ def from_arrow(cls, table):
                     out = out.set_index(idx)
             else:
                 out = out.set_index(index_col[0])
+        if "__index_level_0__" in out.index.names:
```

Comment:
Can we have a test in `cudf` that verifies this code behavior?

Reply:
Added test coverage to cudf